sig-storage/README.md

```diff
@@ -103,7 +103,7 @@ We hold a public meeting every two weeks, on Thursdays at 9 AM (PT).
 * Recordings of past meetings: https://www.youtube.com/playlist?list=PLb1_clREIGYVaqvKaUsVyXxjfcSQDBnmB
 
 ### Contributing
-Interested in contributing to storage features in Kubernetes? [Please read our guide for new contributors](https://github.com/kubernetes/community/blob/master/sig-storage/CONTRIBUTING.md)
+Interested in contributing to storage features in Kubernetes? [Please read our guide for new contributors](https://github.com/kubernetes/community/blob/master/sig-storage/contributing.md)
```

Comment:
While we are here, could you change the link to `/sig-storage/contributing.md` to stay consistent with other links? :grimacing: